cmd/notary/tuf.go

```diff
@@ -546,3 +547,19 @@ func getRemoteTrustServer(config *viper.Viper) string {
 	}
 	return defaultServerURL
 }
+
+func getTrustPinning(config *viper.Viper) notary.TrustPinConfig {
+	// Need to parse out Certs section from config
+	certMap := config.GetStringMap("trust_pinning.certs")
+	resultCertMap := make(map[string][]string)
+	for gun, certSlice := range certMap {
+		if castedCertSlice, ok := certSlice.([]string); ok {
+			resultCertMap[gun] = castedCertSlice
```

Comment:
Non-blocking - wondering if we should error if it's not a slice, so that users will figure out why things aren't getting pinned correctly sooner?